blueocean-web/src/main/js/main.jsx

```diff
@@ -26,7 +26,7 @@ class App extends Component {
             <div className="Site">
                 <div id="outer">
                     <header className="global-header">
-                        <ExtensionPoint name="jenkins.logo.top"/>
+                        <Extensions.Renderer name="jenkins.logo.top"/>
```

Comment:
This is a pretty huge naming change. The term "ExtensionPoint" is pretty well known. It is it still used in the yaml for example to specify where an extension plugs in. But it seems with this there is no more ExtensionPoint? just a Renderer (or whatever type it is). 

I don't think this will be very clear for people scanning the code. Perhaps a better name would be Extensions.RendererExtensionPoint or something? ExtensionPoint.Renderer or something? 

"Extensions.Renderer" is wrong - as Renderer is not an Extension (singular) - but a type of extension point right?